### PR TITLE
Call to action

### PR DESCRIPTION
**Pull request by hicondron**
create html and css rough draft for Call To Action

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
+    <link rel="stylesheet" type="text/css" href="css/style.css"> 
+    
     <title>Investment</title>
 </head>
 <body>
     <section id="wrapper">
 
         <!-- HEADER SECTION -->
-        <header>hi</header>
+        <header></header>
```
Comment (hicondron) [inline index.html:15]: took out header h1 hi for presentation purposes.

File: index.html [hunk 2]
```diff
@@ -4,14 +4,15 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" src="css/style.css"> 
+    <link rel="stylesheet" type="text/css" href="css/style.css">
```
Comment (hicondron) [inline index.html:7]: changed src to href for style sheet work on chrome

File: index.html [hunk 3]
```diff
@@ -20,8 +21,13 @@
         <section id="features"></section>
 
         <!-- CALL TO ACTION SECTION -->
-        <section id="action"></section>
-
+        <section id="action">
```
Comment (hicondron) [inline index.html:24]: add call to action section

**Review by hicondron — COMMENTED** [review]
-changed src to href in linked stylesheet to get css working on chrome
-deleted the h1 "hi" for presentation purposes
-added call to action section

**Review by BachAmadou — APPROVED** [review]

File: css/style.css [hunk 4]
```diff
@@ -9,4 +9,18 @@ html {
 
 body {
     background:red;
```
Comment (hicondron) [inline css/style.css:11]: add basic css styling

**Review by hicondron — COMMENTED** [review]
left comments on css conflicts

**Review by BachAmadou — APPROVED** [review]

**Pull request merged**